bin/anvi-gen-gene-level-stats-databases

```diff
@@ -128,8 +133,13 @@ if __name__ == '__main__':
     groupD = parser.add_argument_group('PARAMETERS OF CONVENIENCE', "They say they save lives.")
     groupD.add_argument(*anvio.A('just-do-it'), **anvio.K('just-do-it'))
 
+    groupE = parser.add_argument_group('INSEQ DATA', "When analyzing INSeq/Tn-Seq data")
+    groupE.add_argument("--inseq-stats", action="store_true")
```

Comment:
These parameters need to be moved to `anvio/__init__.py`, so you can use the same help_text in multiple programs as you do in here and `anvi-interactive`.